opm/input/eclipse/Schedule/Network/NetworkKeywordHandlers.cpp

```diff
@@ -78,12 +78,13 @@ void handleGRUPNET(HandlerContext& handlerContext)
         throw OpmInputError(msg, handlerContext.keyword.location());
     }
     network.set_standard_network(true);
-    const char* info_msg =
-        " \n"
-        " PLEASE NOTE: \n"
-        "   Flow writes restart data for standard network in extended network format. \n"
-        "   Restarting other simulators from Flow output requires conversion to extended network. \n"
-        ;
+    const std::string info_msg = fmt::format(
```

Comment:
no formatting here?

Reply:
not sure what you are referring to.

Reply:
There is no reason to pass a string without any {} through fmt::format.

Reply:
> There is no reason to pass a string without any {} through fmt::format.

That's **almost** true.  We need some kind of formatting in order to interpret the newline escape sequences (`\n`).

Reply:
Sure. As shown in the comment below, I was struggling a little bit with the indentation of the code.  I will do this then. 

```c++

    const std::string info_msg = {
                "\n"
                " PLEASE NOTE:\n"
                "   Flow writes restart data for standard network in extended network format.\n"
                "   Restarting other simulators from Flow output requires conversion to extended network.\n"};
```